docs/source/policies/security_response_policy.rst

```diff
@@ -0,0 +1,28 @@
+NI LinuxRT Security Response Policy
+===================================
+
+NI is committed to resolving major security vulnerabilities discovered in the x64 NI LinuxRT distribution images and core package feeds.
+
+The LinuxRT development team regularly monitors the `NIST National Vulnerability Database <https://nvd.nist.gov/>`_ for published vulnerabilities, and uses that information to audit the x64 NILRT distribution software. When a vulnerability is discovered, the team evaluates its impact to our products and our customers' supported workflows, and assigns the vulnerability a vendor severity score. In many cases, NI's vendor score agrees with the NIST-assigned score. But the team may downgrade a vulnerability if it does not affect the system in a supported configuration.
+
+Vulnerabilities which the team assesses as being high-severity are patched or otherwise fixed in the next release of the distribution. Critical-severity vulnerabilities may have their fixes backported to the latest distribution release, if it is similarly affected.
```

Comment:
Maybe I'm overthinking this, but "latest distribution release" seems potentially ambiguous. If we ship a new release with a certain vuln fixed, would we fix the previous one even though the new release is now the "latest?"

Reply:
That was the best wording I could think of at the time. The alternative I considered was calling it the "current" release. But I think that really has the same ambiguity issues as the releases process. Do you have another alternative in mind?

Reply:
Hmm, perhaps this?

```suggestion
Vulnerabilities which the team assesses as being high-severity will be patched or otherwise fixed in the following release of the distribution. Critical-severity vulnerabilities may be addressed in existing releases, before a scheduled release.
```

Reply:
Fixed Patch V2.